packages/shopping/src/application.ts

```diff
@@ -99,4 +101,15 @@ export class ShoppingApplication extends BootMixin(
 
     this.bind(UserServiceBindings.USER_SERVICE).toClass(MyUserService);
   }
+
+  modifyApiSpecAndRebind() {
+    let openAPISchema = this.getSync(RestBindings.API_SPEC);
+    addSecuritySchemesToOpenApiSpec(openAPISchema);
+    this.bind(RestBindings.API_SPEC).to(openAPISchema);
```

Comment:
No need to rebind if the update to open api spec is in place.

Reply:
This code will be obsolete in my next commit; based on our conversation yesterday.